ol-keycloak/ol-spi/src/main/java/edu/mit/keycloak/forms/login/freemarker/models/OLSettingsBean.java

```diff
@@ -3,19 +3,37 @@
 import java.util.Optional;
 
 import org.keycloak.models.RealmModel;
+import org.apache.http.client.utils.URIBuilder;
 
 import edu.mit.keycloak.util.OLAttributeKeys;
 
+import org.jboss.logging.Logger;
 
 public class OLSettingsBean {
 
+    public static Logger LOGGER = Logger.getLogger(OLSettingsBean.class);
+    public static String HOME_URL = "#";
+
     private String homeUrl;
+    private String privacyPolicyUrl;
 
     public OLSettingsBean(RealmModel realm) {
-        this.homeUrl = Optional.ofNullable(realm.getAttribute(OLAttributeKeys.HOME_URL)).orElse("#");
+        this.homeUrl = Optional.ofNullable(realm.getAttribute(OLAttributeKeys.HOME_URL)).orElse(HOME_URL);
+
+
+        try {
+            this.privacyPolicyUrl = new URIBuilder(this.homeUrl).setPath("/privacy").toString();
+
+        } catch (Exception e) {
+            this.privacyPolicyUrl = HOME_URL;
+        }
```

Comment:
Doesn't really matter to me, but I think in Java, the use of exceptions for flow control is an anti-pattern.

Reply:
`URIBuilder` throws an exception so I need to handle it or it won't compile.

Reply:
It might be good to indicate what would cause the exception, or build a conditional to circumvent the exception.  Right now, the logic for determining how this.privacyPolicyUrl is defined is a bit of mystery to me